content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+### Step 6: Add files to the repository
 
- * `disqusShortname` is for activating comment section on a blog page or other page which needs a comment section. get your Disqus short name from [here](https://disqus.com/). if you don't need the comment section, then leave it empty.
+Now your site is ready to run. Add a file with the name `.gitignore` to your root directory and fill it with the contents of [this autogenerated sample file](https://www.toptal.com/developers/gitignore/api/node,go,hugo). This will exempt several files from being added to the repository that are created automatically. You can change the list of supported programming languages and IDEs [at this link](https://www.toptal.com/developers/gitignore?templates=node,go,hugo).
 
- * `DefaultContentLanguage` field is for your site default language, which will load first with your site load.
+After that add all files to your repository and commit.
 
- * `disableLanguages` field is for disabling languages, if you don't want to show one or more then one language then you can add the language name here, it will disable those languages from your site. example `disableLanguages = ["fr"]` it will disable french language from your site.
+```bash
+git add -A
+git commit -m "first commit of all website files"
+```
 
-#### Default Parameters
+### Step 7: Test your website
 
-Default parameters are those defined by the theme developer. Default parameters start with `[params]` heading.
+Hugo comes with an integrated development server. You can easily start the server with the following command:
 
- * `logo` is for your site logo, give the logo directory here (i.e. `logo = images/logo.png`). Logo supports `png`, `jpg`, `gif`, or `svg` format. If you leave it empty, then `title` (default configuration title) will replace the logo.
+```shell script
+hugo server
+```
 
- * `footer_logo` is for your footer logo. Most of the themes don't need that. It's only need when the footer logo is different color.
+Hugo will load everything and start the server and let you know at which URL you can load the test site. Open your browser and enjoy your new shiny Hugo website.
 
- * `favicon` is defined in your theme header. So you don't need to specify the directory here. Just put your favicon in `static/images` folder. Remind that you need to put this favicon name `favicon.png`. Otherwise, it won't load. If the theme already has a favicon, then replace it with your new one. It will be shown on the browser tab when you compiled the theme again. If it's not showing, then you need to clear your browser cache and reload again.
+## Local development in a nutshell
 
- * `description` is for your default meta description, it will show on your homepage and other pages that have not set any meta description on this page front-matter.
+{{< youtube dyZcRRWiuuw >}}
 
- * `author` is also a meta tag. It belongs to the site author.
+**If the list above sounds like a foreign language to you, that’s okay! Let’s break it down:**
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)